src/BuildScriptGenerator/PythonVersions.cs

```diff
@@ -15,6 +15,7 @@ public static class PythonVersions
         public const string Python39Version = "3.9.15";
         public const string Python310Version = "3.10.8";
         public const string Python311Version = "3.11.0";
+        public const string Python312Version = "3.12.0";
```

Comment:
Could we add tests for 3.12? You could reference what we're doing for `Python311Version`.

Reply:
yah we should. I will add the test. Thanks.